.scripts/ci.yml

```diff
@@ -228,6 +231,25 @@ stages:
             workingDirectory: $(Build.SourcesDirectory)/packages/typespec-test
             displayName: "Check git Tree"
 
+      - job: TypeSpec_Environment_Check
+        dependsOn: Build
+        variables:
+          - template: globals.yml
+        strategy:
+          matrix:
+            Windows_Latest:
```

Comment:
Only enable the windows environemnt and linux is already covered in complete smoke-test.